src/Common/src/TypeSystem/IL/ILReader.cs

```diff
@@ -0,0 +1,119 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Internal.TypeSystem;
+
+namespace Internal.IL
+{
+    //
+    // Corresponds to "I.12.3.2.1 The evaluation stack" in ECMA spec
+    //
+    internal enum StackValueKind
```

Comment:
Move this out to a separate shared file `StackValueKind.cs` since it looks exactly the same as the one in ILImporter.cs ?

Reply:
Done!